src/plugins/elasticsearch/index.js

```diff
@@ -1,38 +1,38 @@
-var _ = require('lodash');
-let Boom = require('boom');
+import { trim, trimRight } from 'lodash';
+import { methodNotAllowed } from 'boom';
 
-module.exports = function (kibana) {
-  var healthCheck = require('./lib/health_check');
-  var exposeClient = require('./lib/expose_client');
-  var createProxy = require('./lib/create_proxy');
+import healthCheck from './lib/health_check';
+import exposeClient from './lib/expose_client';
+import createProxy, { createPath } from './lib/create_proxy';
 
-  return new kibana.Plugin({
+module.exports = function ({ Plugin }) {
```

Comment:
Since we agreed that we'll do a bulk change of commonjs modules for 5.0, I'd prefer to leave this commonjs export in place in this PR for now.